libs/pruviloj/Pruviloj/Core.idr

```diff
@@ -268,3 +274,37 @@ reflexivity =
                 , NamePart `{reflexivity}
                 , TextPart "is not applicable."
                 ]
+
+||| Attempt to swap the sides of equality in a goal.
+symmetry : Elab ()
+symmetry = do
+    [_,_,_,_,_] <- apply (Var `{{sym}}) [True, True, True, True, False]
+       | _ => fail [TextPart "Failed while applying", NamePart `{symmetry} ]
+    solve
+    attack
+
+||| Swap the sides of an equality and bind the result to supplied name.
+symmetryIn : Raw -> TTName -> Elab ()
+symmetryIn t n = do
```

Comment:
It is interesting that this and `symmetry` are independent.
Do you know if it is possible to implement them in terms of each other, or do they just share name?